rest/src/main/java/org/jboss/pnc/rest/endpoint/UserEndpoint.java

```diff
@@ -193,4 +193,22 @@ public Response update(@ApiParam(value = "User id", required = true) @PathParam(
        return super.update(id, userRest);
     }
 
+    @ApiOperation(value = "Get all BuildRecords (running and archived) triggered by this User, returns empty list if no build records are found")
```

Comment:
To be removed after testing, will be available via `pnc-rest/rest/builds?q=(user.id==1)`